doc/_static/petab_schema.yaml

```diff
@@ -78,8 +91,12 @@ properties:
             type: string
             description: PEtab visualization file name or URL.
 
+        mapping_file:
```

Comment:
For all other file types we now allow for multiple files. It would be more coherent to also turn this into a list of filenames. Opinions?

Reply:
Agreed!